tests/test_text/test_dispersion.py

```diff
@@ -21,23 +21,28 @@
 import pytest
 
 from yellowbrick.exceptions import YellowbrickValueError
+from yellowbrick.datasets import load_hobbies
 from yellowbrick.text.dispersion import *
-from tests.dataset import DatasetMixin
 from tests.base import VisualTestCase
 import matplotlib.pyplot as plt
 
+##########################################################################
+## Data
+##########################################################################
+
+corpus = load_hobbies()
```

Comment:
Just out of curiosity, why make this variable a module level var? The reason I ask is that typically when I see this, there is some advantage to be gained by caching expensive work across all tests in the module, but I want to make sure we're on the same page in that the data is read from disk (e.g. not cached) every time `corpus.data` is called. This is not true for `corpus.labels`, which is cached on the first read. 

The `Corpus` object cannot be easily modified because of it's use `@property` decorators, so I'm not worried about testing errors due to `corpus` being modified between different tests, and so placing this here will save some lines of code in the tests below.

Reply:
I don't have a strong opinion about how this should be done, and I couldn't find an example of how we were planning to use the datasets as specified in #682. The closest I could find were things like [this](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/tests/test_classifier/test_class_prediction_error.py#L39), [this](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/tests/test_cluster/test_elbow.py#L49), and [this](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/tests/test_features/test_jointplot.py#L58). Would you like to propose a default method for data fixtures (e.g. just load the data for each test) that can be used across all the updated tests? I can update the issue with that spec and then as I go through an update more of the tests, I'll aim to make everything consistent.

Reply:
Ok, I just wanted to make sure it wasn't there for some reason I didn't understand - it's perfectly fine being there! 

As for a default method, I think that's going to have to be a little nuanced. The first two examples you gave both have the same caveats like the ones I mentioned above, and I'm not a fan of the middle example, I'd prefer to use `make_classification` as in the first example. The third example is very different, it is defining a function and every time `rand1d()` is called a new dataset is generated (`partial` assigns default arguments to the original function). This is sort of a strange example because in that case the tests are relying on the randomness rather than setting a fixed random state. 

The safest thing to do is to use a [fixture](https://docs.pytest.org/en/latest/fixture.html) as in the [binary classification fixture](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/tests/test_classifier/conftest.py#L30) and [here](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/tests/test_classifier/conftest.py#L30), but that may simply be adding an extra layer of obscurity that prevents new contributors from understanding our tests. 

For now, this looks good and I think we should merge the PR without changes!